.github/workflows/ci.yml

```diff
@@ -16,8 +16,12 @@ jobs:
         run: |
           # We are not yet using the new naming in the deploy workflow
           docker build -t gitbot:latest .
-          docker run -p 8080:8080 -d -e GOOGLE_APPLICATION_CREDENTIALS -e FAST_STARTUP=1 gitbot:latest
```

Comment:
`GOOGLE_APPLICATION_CREDENTIALS` may have been required in the past but it has no effect.